core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpConnectionContext.java

```diff
@@ -168,22 +167,22 @@ LineTcpConnectionContext of(long clientFd, IODispatcher<LineTcpConnectionContext
     protected final IOContextResult parseMeasurements(NetworkIOJob netIoJob) {
         while (true) {
             try {
-                ParseResult rc = goodMeasurement ? protoParser.parseMeasurement(recvBufPos) : protoParser.skipMeasurement(recvBufPos);
+                ParseResult rc = goodMeasurement ? parser.parseMeasurement(recvBufPos) : parser.skipMeasurement(recvBufPos);
                 switch (rc) {
                     case MEASUREMENT_COMPLETE: {
                         if (goodMeasurement) {
-                            if (scheduler.tryButCouldNotCommit(netIoJob, protoParser, charSink)) {
+                            if (scheduler.scheduleEvent(netIoJob, parser, floatingDirectCharSink)) {
```

Comment:
I cannot believe this cryptic name will be done `tryButCouldNotCommit`

Reply:
me neither!